pkg/config/config_template.yaml

```diff
@@ -1013,12 +1013,18 @@ api_key:
   #
   # max_message_size_bytes: 256000
 
-  ## @param integrations_logs_files_max_size - integer - optional - default: 100
-  ## @env DD_LOGS_CONFIG_INTEGRATIONS_LOGS_FILES_MAX_SIZE - integer - optional - default: 100
-  ## The combined size in MB of all the integration logs files the Agent is allowed to write.
+  ## @param integrations_logs_files_max_size - integer - optional - default: 10
+  ## @env DD_LOGS_CONFIG_INTEGRATIONS_LOGS_FILES_MAX_SIZE - integer - optional - default: 10
+  ## The max size in MB that an integration logs file is allowed to use
   #
   # integrations_logs_files_max_size
```

Comment:
Would reducing the default by a factor of 10 not break backward compatibility ?

Reply:
Normally yes, but the feature this applies to hasn't been released to GA yet.